### PR TITLE
Docker file

### PR DESCRIPTION
**Pull request by utarn**

### CODE REVIEW

File: content/faq/dotnet/dotnet-cli.md [hunk 1]
```diff
@@ -31,3 +31,48 @@ notepad ./global.json
...
+dotnet new mvc
+```
+
+โปรแกรม dotnet cli จะสร้างเฉพาะตัวอย่างของเว็บ MVC ที่ไม่มีการ Authentication หากต้องการระบุให้มีการ Authentication สามารถใช้คำสั่งด้วยพารามิเตอร์ --auth เช่น
+
+```sh
+dotnet new mvc --auth Individual
+```
+
+จากตัวอย่างคำสั่ง จะสร้างระบบ Authentication ด้วย ASP.NET Identity ที่ใช้ local database ซึ่งหากสร้างโปรเจคนี้ใน Visual Studio โปรแกรม Visual Studio จะเลือกใช้ฐานข้อมูล SQL Server LocalDB ให้อัตโนมัติ แต่ว่าหากพิมพ์คำสั่งนี้ผ่าน dotnet cli โปรแกมรจะเลือกใช้ฐานข้อมูล SQLLite ให้อัตโนมัติ ดังนั้น ในกรณีที่ต้องการใช้ SQL Server LocalDB ผู้อ่านสามารถใช้คำสั่ง
```
Comment (dmakeroam) [inline content/faq/dotnet/dotnet-cli.md:50]: fix: typo: โปรแกรม

File: content/faq/dotnet/dotnet-cli.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
```
Comment (dmakeroam) [inline content/faq/dotnet/dotnet-cli.md]: fix: typo: โปรแกรม

**Review by dmakeroam — CHANGES_REQUESTED** [review]

**Comment by utarn** [comment]
sorry for the late fix.

**Comment by aaronamm** [comment]
@utarn Thanks for your contribute. @dmakeroam will continue review your PR and he will merge it if all PR feedbacks have been fixed.
Thanks.

**Pull request merged**